### PR TITLE
chore(github): add required code owner sign-off for packages

### PR DESCRIPTION
**Pull request by chrisdholt**
<body>

<footer>
 

For [details on formatting](https://github.com/Microsoft/fast-dna/wiki/pull-request-guidance) pull requests.

### CODE REVIEW

File: .github/CODEOWNERS.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners,
+# the last matching pattern has the most precendence.
+
+# These owners will be the default owners for everything in the repo.
+*       @chrisdholt @janechu @nicholasrice
+
+# File type specific owners
+
+# .md specific files
```
Comment (awentzel) [inline .github/CODEOWNERS.md]: Perhaps change this to "Markdown files"

File: .github/CODEOWNERS.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+# Code Owners
+# Code owners have the ability to approve a pull request that has made changes in the below files or package(s).
```
Comment (awentzel) [inline .github/CODEOWNERS.md]: Generally, I try to avoid repeating the title as the first words of the first sentences. It's a nit, but, it reads awkward to me.  Suggest rewording.

**Comment by chrisdholt** [comment]
@awentzel It's not a .md file (so I have removed .md from the file type). The "#" are comments. This should fix the issue with reading in the .md reader

File: .github/CODEOWNERS.md (hunk 1 above)
Comment (chrisdholt) [reply .github/CODEOWNERS.md]: Updated

File: .github/CODEOWNERS.md (hunk 2 above)
Comment (chrisdholt) [reply .github/CODEOWNERS.md]: Removed the title in favor of the paragraph
